src/Language/Haskell/Liquid/Types.hs

```diff
@@ -458,8 +458,9 @@ instance Hashable (PVar a) where
 ------ Strictness --------------------------------------------------
 --------------------------------------------------------------------
 
-instance NFData Var
-instance NFData SrcSpan
+instance NFData Var where rnf _ = ()
```

Comment:
I disagree, I don't find Werror to be particularly useful in Haskell where most of the warnings are cosmetic or stylistic. 

In contrast, not everyone will rush to upgrade to 7.10; so moving to 7.10 will prevent people from using LH.